### ISSUE TITLE
Extension failed to find deno

### ISSUE THREAD

**Issue by rico345100** [ameerthehacker/deno-vscode#5]
Hello!

I installed this extension, but when I run "deno init", it gives me this error:

> Could not find deno in the path, please ensure that you have installed deno and it is available in the path

Of course, I installed deno. How do I fix this? Thanks!

**Comment by ameerthehacker** [comment]
Hi @rico345100 did you add deno to the system path and are you able to access deno from commandline?

**Comment by ameerthehacker** [comment]
@rico345100 closing this issue due to no activity

**Comment by euphbriggs** [comment]
I had a similar error. I'm running VS Code on Windows and using the "Remote - WSL" plugin to access Windows Subsystem for Linux.

For me, when I opened a terminal and typed `deno version` I would get the expected output, but the plugin was unable to run it. Your suggestion to add it to the system path worked for me. I added a symlink to `deno` into my system path and restarted VS Code. It picked it up from there. Thanks!

My guess is VS Code runs the remote connection in a different environment and it didn't pick up my user's PATH.

Without more information, I'm not sure if this is what the OP was experiencing, but I thought I'd leave some feedback that it helped me.
